### PR TITLE
Describe how to distinguish DTLS 1.2 and DTLS 1.3 connections. Fixes …

### PR DESCRIPTION
**Pull request by ekr**
…#159

### CODE REVIEW

File: draft-ietf-tls-dtls13.md [hunk 1]
```diff
@@ -367,7 +367,11 @@ The DTLSCiphertext header is tightly bit-packed, as shown below:
 Fixed Bits:
 : The three high bits of the first byte of the DTLSCiphertext header are set to
   001. This ensures that the value will fit within the DTLS region when
-  multiplexing is performed as described in {{?RFC7983}}.
+  multiplexing is performed as described in {{?RFC7983}}. It also ensures
+  that it is possible to distinguish encrypted DTLS 1.3 records from
+  encrypted DTLS 1.2 records when they are carried on the same host/port quartet;
```
Comment (kaduk) [inline draft-ietf-tls-dtls13.md:372]: (Do we care about "host/port quartet" vs. "5-tuple"?  I don't think I do...)

**Review by kaduk — APPROVED** [review]
This WFM

**Comment by ekr** [comment]
I don't.

File: draft-ietf-tls-dtls13.md (hunk 1 above)
Comment (chris-wood) [inline draft-ietf-tls-dtls13.md:372]:
```suggestion
  multiplexing is performed as described in {{?RFC7983}}. It also ensures
  that distinguishing encrypted DTLS 1.3 records from encrypted DTLS 1.2 
  records is possible when they are carried on the same host/port quartet;
```

**Review by chris-wood — APPROVED** [review]

**Pull request merged**